{"name": "加购物车", "status": "broken", "statusDetails": {"message": "TypeError: 'dict' object is not callable", "trace": "self = <testcases.usercenter.test_user.TestUser object at 0x10ab4bac0>\nusername = 17380900323, password = 123456\n\n    @allure.story(\"购物车相关\")\n    @allure.title(\"加购物车\")\n    @pytest.mark.parametrize(\"username,password\", base_data.read_data()['user_login'])\n    def test_shopping_cart(self, username, password):\n        # 登录接口\n        result = login(username, password)\n        token = result.body['token']\n>       param = get_data()['shopping_cart']\nE       TypeError: 'dict' object is not callable\n\ntest_user.py:48: TypeError"}, "attachments": [{"name": "log", "source": "1d5745ca-b796-441e-88f9-2e15c7757a41-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "ed340baa-35a4-423b-b317-96e77d1ed3c1-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "17380900323"}, {"name": "password", "value": "123456"}], "start": 1659157944056, "stop": 1659157944217, "uuid": "e2198780-eec2-412a-80f3-4c83d7320256", "historyId": "078982f37d6fc574e848a2dd911995ae", "testCaseId": "6732b5a4ef5cebf11d6a36876a07be7c", "fullName": "testcases.usercenter.test_user.TestUser#test_shopping_cart", "labels": [{"name": "feature", "value": "用户中心模块"}, {"name": "story", "value": "购物车相关"}, {"name": "parentSuite", "value": "testcases.usercenter"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "shallowdeMacBook-Air.local"}, {"name": "thread", "value": "60864-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.usercenter.test_user"}]}